app/sesame/engine/recent_valkey_test.go

```
@@ -0,0 +1,301 @@
+	ln net.Listener
+
+	mu        sync.Mutex
+	members   []string
+	lastRange []string
+}
+
+func newRecentScriptedServer(tb testing.TB) *recentScriptedServer {
+	tb.Helper()
+	ln, err := net.Listen("tcp", "127.0.0.1:0")
+	require.NoError(tb, err)
+	tb.Cleanup(func() { ln.Close() })
```

golangci-lint (2.12.2)

[error] Error return value of `ln.Close` is not checked

(errcheck)